src/test/regress/specs/isolation_create_restore_point.spec

```diff
@@ -32,6 +34,15 @@ step "s1-modify-multiple"
 	SELECT master_modify_multiple_shards($$UPDATE restore_table SET data = 'world'$$);
 }
 
+step "s1-multi-statement"
+{
+	SET citus.multi_shard_commit_protocol TO '2pc';
+	BEGIN;
+	INSERT INTO restore_table VALUES (1,'hello');
+	INSERT INTO restore_table VALUES (2,'hello');
+	COMMIT;
+}
+
```

Comment:
Would it make sense to add a multi shard update/delete (not `master_modify_multiple_shards`) case as well?